test/config/getconfig_test.cc

```diff
@@ -137,7 +137,29 @@ TEST_F(GetConfigTest, ReturnsTheConfig) {
   ASSERT_EQ(oidc.authorization_uri(), "https://google3/path3");
   ASSERT_EQ(oidc.token_uri(), "https://google2/path2");
   ASSERT_EQ(oidc.callback_uri(), "https://google4/path4");
-  ASSERT_EQ(oidc.jwks(), "jwks_placeholder");
+  ASSERT_EQ(
+      oidc.jwks(),
+      "{\n  \"keys\": [\n    {\n      \"e\": \"AQAB\",\n      \"kty\": "
```

Comment:
you can use multi line string literal for easier editing.
https://stackoverflow.com/a/5460235